epictrack-api/migrations/versions/8e4bb9cd046c_issue_and_status.py

```diff
@@ -0,0 +1,178 @@
+"""issue and status
+
+Revision ID: 8e4bb9cd046c
+Revises: f9bcc9a0564b
+Create Date: 2023-11-02 09:00:55.167458
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '8e4bb9cd046c'
+down_revision = 'f9bcc9a0564b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('work_issues',
```

Comment:
should table name be plural?

Reply:
no standard conventions..but easier to go with plural